river/tree/splitter/base_splitter.py

```diff
@@ -1,11 +1,17 @@
 from abc import ABCMeta, abstractmethod
 
+from river import base
+
 from .._attribute_test import AttributeSplitSuggestion
 
 
-class AttributeObserver(metaclass=ABCMeta):
-    """Abstract class for observing the class data distribution for an attribute.
-    This observer monitors the class distribution of a given attribute.
+class Splitter(base.Estimator, metaclass=ABCMeta):
+    """Base class for the tree splitters.
+
+    Attribute Observers (AO) or Splitters are responsible by monitoring an input feature
+    and evaluating possible split points to them. They can also perform other tasks related
```

Comment:
Review that

Reply:
Changed in commit [d6931d9](https://github.com/online-ml/river/pull/488/commits/d6931d9e1632978aeb7276b9a13cff1ac2a3ec09).